atomic_reactor/plugins/pre_resolve_remote_source.py

```diff
@@ -76,84 +78,70 @@ def run(self):
             return
 
         if (not get_allow_multiple_remote_sources(self.workflow)
-                and self.workflow.source.config.remote_sources):
+                and self.multiple_remote_sources_params):
             raise ValueError('Multiple remote sources are not supported, use single '
```

Comment:
instead of 'not supported' => 'not allowed' or 'not enabled'

Reply:
fixed